cea/datamanagement/format_helper/cea4_verify.py

```
@@ -0,0 +1,402 @@
+              'If you manually prepared the input data, check the log for missing files and/or incompatible columns. Modify your input data according to the log above.'
+              )
+
+
+## --------------------------------------------------------------------------------------------------------------------
+## Unique traits for the CEA-4 format
+## --------------------------------------------------------------------------------------------------------------------
+
+def cea4_verify(scenario, print_results=False):
+
+    #0. get the scenario name
+    scenario_name = os.path.basename(scenario)
```

Ruff (0.8.2)

Local variable `scenario_name` is assigned to but never used

Remove assignment to unused variable `scenario_name`

(F841)